hms-app/src/components/card-details/HackathonDetails.tsx

```diff
@@ -297,9 +300,12 @@ export default function HackathonDetails(props: IProps) {
 
             {type === HackathonDetailsType.Archive && hackathonId !== '' && (
               <Container mb={25}>
-                <RichTextEditor readOnly value={value!} onChange={onChange}>
-                  {hackathonData.description}
-                </RichTextEditor>
+                <RichTextEditor
+                  readOnly
+                  value={hackathonData.description || ''}
+                  onChange={onChange}
+                  id='rte'
+                />
```

Comment:
this should probably use another option as well with your `initialValue` constant, something like.

```suggestion
                <RichTextEditor
                  readOnly
                  value={hackathonData.description || ''}
                  onChange={onChange}
                  placeholder={intialValue}
                  id='rte'
                />
```

I'm not sure if this extension is part of mantine's RichTextEditor by default, but I saw examples of it's use in their docs

Reply:
so this should apply to the `HackathonForm.tsx` file and not this one since you should never need a Placeholder because there should be a value here or nothing